app/lib/search/search_service.dart

```diff
@@ -53,6 +53,8 @@ class PackageDocument {
 
   final List<String> platforms;
 
+  final List<String> tags;
```

Comment:
If this can be `null` during transition, I propose that we document that this might be `null` as we are upgrading the index.
Leaving a TODO that all `null` handling should be removed in the future.